routes/api/v1.js

```diff
@@ -439,20 +440,20 @@ api.declare({
     "Get temporary AWS credentials for `read-write` or `read-only` access to",
     "a given `bucket` and `prefix` within that bucket.",
     "The `level` parameter can be `read-write` or `read-only` and determines",
-    "which type of credentials is returned. Please note that the `level`",
+    "which type of credentials are returned. Please note that the `level`",
     "parameter is required in the scope guarding access.",
     "",
-    "The credentials are set of expire after an hour, but this behavior may be",
+    "The credentials are set to expire after an hour, but this behavior is",
     "subject to change. Hence, you should always read the `expires` property",
-    "from the response, if you intent to maintain active credentials in your",
+    "from the response, if you intend to maintain active credentials in your",
     "application.",
     "",
-    "Please notice that your `prefix` may not start with slash `/`, it is",
-    "allowed on S3, but we forbid it here to discourage bad behavior.",
-    "Also note that if your `prefix` doesn't end in a slash `/` the STS",
-    "credentials will not require one to be to inserted. This is mainly a",
+    "Please note that your `prefix` may not start with slash `/`. Such a prefix",
+    "is allowed on S3, but we forbid it here to discourage bad behavior.",
+    "Also note that if your `prefix` doesn't end in a slash `/`, the STS",
+    "credentials will not require a trailing slash. This is mainly a",
```

Comment:
I think we're saying the same thing - "one will not be required to be inserted" is a double-passive form of "there is no need to insert one", where "one" refers to a trailing slash.  But I'll try to say it a different way.